plugins/inventory/iocage.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024 Vladimir Botka <vbotka@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: iocage
+    short_description: iocage inventory source
+    version_added: 10.2.0
+    author:
+        - Vladimir Botka (@vbotka)
+    requirements:
+        - iocage >= 1.8
+    description:
+        - Get inventory hosts from the iocage jail manager.
+        - Uses a configuration file as an inventory source, it must end
+          in C(.iocage.yml) or C(.iocage.yaml).
```

Comment:
I think description should mention that remote hosts are contacted via SSH. From the description I would assume that the plugin simply runs iocage locally.

Reply:
OK. I added an item to the description.